masterlist.yaml

```
@@ -22625,3 +22625,13 @@ plugins:
         crc: 0xDC20C34C
         util: '[SSEEdit v4.0.4b](https://www.nexusmods.com/skyrimspecialedition/mods/164)'
         itm: 2
+
+  - name: 'MarkekrausSentientChairsOfSkyrim.esp'
+    url: [ 'https://www.nexusmods.com/skyrimspecialedition/mods/59604' ]
```

Please add a trailing slash for consistency:

```suggestion
    url: [ 'https://www.nexusmods.com/skyrimspecialedition/mods/59604/' ]
```